data/secscan_model/__init__.py

```diff
@@ -18,22 +18,28 @@
 class SecurityScannerModelProxy(SecurityScannerInterface):
     def configure(self, app, instance_keys, storage):
         # TODO(alecmerdler): Just use `V4SecurityScanner` once Clair V2 is removed.
-        self._model = V2SecurityScanner(app, instance_keys, storage)
-        self._v4_model = V4SecurityScanner(app, instance_keys, storage)
+        try:
+            self._model = V2SecurityScanner(app, instance_keys, storage)
+        except Exception:
```

Comment:
I'd prefer if we caught something more specific than just general `Exception` here